Makefile

```diff
@@ -1,33 +1,61 @@
 COQMODULE    := OpencypherCoq
 COQTHEORIES  := src/*.v
 
-.PHONY: all theories clean tounicode
+.PHONY: all theories fmt lint docs clean tounicode
 
 all: build
 
 build: Makefile.coq
+	eval $$(opam env) && \
```

Comment:
Why have you added it here? This is strange that the build changes the global shell environment.

Reply:
Fixed this. User is expected to modify their own environment themselves. This is now also done in CI.